### PR TITLE
(#3617) - Support for new SqlitePlugin params

### PR DESCRIPTION
**Pull request by nolanlawson**
@tannerlinsley this is my suggested version of your PR, with a bit of code cleanup to better match our style and jshint, as well as some documentation. Can you use this in your app and verify that it works the same? Also please let me know if the documentation makes sense; I am not an iOS expert. :P

### CODE REVIEW

File: docs/_includes/api/create_database.html [hunk 1]
```diff
@@ -8,20 +8,34 @@
...
+* `ajax.username` + `ajax.password`: You can specify HTTP auth parameters either by using a database with a name in the form `http://user:pass@host/name` or via the `username` + `password` options.
+  
+**WebSQL-only options:**
+
+* `size`: amount in MB to request for storage, which you will need if you are storing >5MB in order to avoid storage limit errors on iOS/Safari. Details [here](/errors.html#not_enough_space).
+
+**SQLite Plugin-only options:**
+
+* `location`: where to store data on iOS, which may affect iTunes/iCloud backup, and thus whether or not your app gets rejected by Apple. See [SQLite Plugin documentation](https://github.com/brodysoft/Cordova-SQLitePlugin/#opening-a-database) and [iOS data storage guidelines](https://developer.apple.com/icloud/documentation/data-storage/index.html).
+* `createFromLocation`: use a pre-populated database, so you can package it with your app and users don't have to wait for it to load. See the [SQLite Plugin documentation](https://github.com/brodysoft/Cordova-SQLitePlugin/#pre-populated-database).
```
Comment (nolanlawson) [inline docs/_includes/api/create_database.html]: Lordy lordy, our options are so complicated...

**Comment by nolanlawson** [comment]
Tiny bug, accidentally used `window.name` again :stuck_out_tongue: . Fixed it...

**Comment by tannerlinsley** [comment]
I'm testing it right now

**Comment by tannerlinsley** [comment]
It works great!  Thanks for the help.

**Comment by nolanlawson** [comment]
No prob. :) Thanks for the contribution.

**Comment by nolanlawson** [comment]
For anyone who is impatient, here are builds of this PR: [pouchdb.js](https://nolanlawson.s3.amazonaws.com/pouchdb/www/20150315/pouchdb.js) and [pouchdb.min.js](https://nolanlawson.s3.amazonaws.com/pouchdb/www/20150315/pouchdb.min.js).

**Comment by tannerlinsley** [comment]
Thanks! I was extremely impatient, so I had to do my own build for my app I
just submitted!

**Comment by daleharvey** [comment]
Awesome, thanks a lot 

https://github.com/pouchdb/pouchdb/commit/82d8897635e118bd03e7d9a3f6b6adeaf053b255

**Pull request closed without merging**